test/DynamoCoreWpfTests/DynamoTestUIBase.cs

```diff
@@ -132,10 +130,10 @@ protected virtual DynamoModel.IStartConfiguration CreateStartConfiguration(IPath
         [TearDown]
         public void Exit()
         {
+          
+            Dispatcher.CurrentDispatcher.Hooks.OperationPosted -= Hooks_OperationPosted;
```

Comment:
should we remove the handler only after the DoEventsLoop ?
I wonder if operations can still post other operations..

Reply:
interesting...